packages/Babylonian-Core.package/BPTrace.class/instance/traceValues.st

```diff
@@ -1,8 +1,11 @@
 accessing
 traceValues
-	
+
 	"Reduce does not work for empty collections, so we have to add an extra conditional"
-	self probes values ifEmpty: [^ self probes values].
-	^ (self probes values fold: [:a :b | a , b]) sorted: [:a | a chronologicalPosition] ascending
+	| probesAndAssertions |
+	probesAndAssertions := self probes asCollection.
+	probesAndAssertions addAll: self assertions asCollection.
+	probesAndAssertions ifEmpty: [^ probesAndAssertions values ].
+	^ (probesAndAssertions values fold: [:a :b | a , b]) sorted: [:a | a chronologicalPosition] ascending.
```

Comment:
I just realized that this might simply be `probesAndAssertions flatten sorted: ...`.

Reply:
This would also make the #ifEmpty: send above unnecessary.

Reply:
That's only suitable for `SequenceableCollections`. Code will not be changed and stays as `fold: ...`